README.md

```diff
@@ -0,0 +1,80 @@
+<!--
+  Licensed to the Apache Software Foundation (ASF) under one or more
+  contributor license agreements.  See the NOTICE file distributed with
+  this work for additional information regarding copyright ownership.
+  The ASF licenses this file to You under the Apache License, Version 2.0
+  (the "License"); you may not use this file except in compliance with
+  the License.  You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+-->
+
+# Apache Daffodil SBT Plugin
+
+Plugin to enable Daffodil capabilities for DFDL schema projects.
+
+## Enable
+
+To enable the plugin, add the following to `project/plugins.sbt`:
+
+```scala
+addSbtPlugin("org.apache.daffodil" % "sbt-daffodil" % "<version>")
+```
+
+## Features
+
+### Saved Parsers
+
+This plugin adds the ability to create and publish saved parsers of a schema.
+
+For each saved parser to generate, add an entry to the
+`daffodilPackageBinInfos` setting. This setting is a Seq of 3-tuples made up of
+the resource path to the schema, an optional root element to use in that
```

Comment:
Need to add If the root is not specified what element does it use ... first in the named schema.